packages/react-brand-designer/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@trycourier/react-brand-designer",
+  "version": "1.17.0",
+  "description": "Emendable brand designer for multi-channel notifications",
+  "author": "Drew Youngwerth <drew@youngwerth.com>",
```

Comment:
I'd change this to Courier maybe

Reply:
Good catch. That was a yarn init thing. Removed. (None of the other packages have an author)